### PR TITLE
TC11 - User can't create account while password and PID fields are empty

### PR DESCRIPTION
**Pull request by sattdn21-01-02**

### CODE REVIEW

File: src/test/java/com/logigear/RegisterTest.java [hunk 1]
```diff
@@ -26,4 +26,27 @@ public void TC07() {
         String expectedMsg = "You're here";
         Assert.assertEquals(actualMsg, expectedMsg, "Error message fails to display!");
     }
+
+    @Test(description = "TC11 - User can't create account while password and PID fields are empty")
+    public void TC11() {
+        LoggerHelper.startTestCase("TC11 - User can't create account while password and PID fields are empty");
+
+        String blankPassword = "";
```
Comment (tanmle) [inline src/test/java/com/logigear/RegisterTest.java:34]: again? I already commented about this in this PR: https://github.com/sattdn21-01-02/selenium-minh2/pull/9
and now you do that again? why?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/RegisterTest.java (hunk 1 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/RegisterTest.java:34]: sorry that my fault, I will never repeat  this error again

File: src/test/java/com/logigear/RegisterTest.java [hunk 2]
```diff
@@ -17,13 +17,34 @@
...
+        homePage.goToRegisterPage();
+        Account account = new Account(Constant.USERNAME, "", "");
+        registerPage.register(account);
+
+        String actualErrorMsg = registerPage.getGeneralErrorMessage();
+        String actualErrorPasswordMsg = registerPage.getPasswordErrorMessage();
+        String actualErrorPidMsg = registerPage.getPidErrorMessage();
+
+        Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message fails to display! ");
+        Assert.assertEquals(actualErrorPasswordMsg, expectedErrorPasswordMsg, "Error password message fails to display!");
```
Comment (tanmle) [inline src/test/java/com/logigear/RegisterTest.java:47]: Error password message -> this is Vietnamese to English

File: src/test/java/com/logigear/RegisterTest.java [hunk 3]
```diff
@@ -17,13 +17,34 @@
...
+        Assert.assertEquals(actualErrorPidMsg, expectedErrorPidMsg, "Error pid message fails to display!");
```
Comment (tanmle) [inline src/test/java/com/logigear/RegisterTest.java:48]: same here

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/RegisterTest.java (hunk 2 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/RegisterTest.java:47]: Thank you for the review, I will improve my grammar English every day

**Review by tanmle — APPROVED** [review]

**Pull request merged**